lib/deploy.js

```diff
@@ -32,7 +32,19 @@ module.exports = promisify(function(config, callback) {
   var ghPagesConfig = {};
   var rootDir = config.rootDir ? config.rootDir : '.';
   var cloneDir = config.cloneDir ? config.cloneDir : null;
-  var commitMessage = config.message ? config.message : childProcess.execSync('git log --format=%B -n 1').toString().trim();
+
+  var commitMessage = config.message || (function() {
+    var spawn = childProcess.spawnSync('git', ['log', '--format=%B', '-n', '1']);
+    var errorText = spawn.stderr.toString();
```

Comment:
Nit: It looks like this has a trailing newline character attached to it, so it'd be helpful to `.trim()` it.